projects/ngx-datatable/src/lib/components/header/header.component.ts

```diff
@@ -272,9 +277,9 @@ export class DataTableHeaderComponent implements OnDestroy, OnChanges {
     column.isTarget = false;
     column.targetMarkerContext = undefined;
     this.reorder.emit({
-      column: model,
-      prevValue: prevIndex,
-      newValue: newIndex
+      model,
+      prevIndex,
+      newIndex
```

Comment:
@spike-rabbit as discussed: Let's align this already if we can, let me know if you need anything from my end 🙇